### ISSUE TITLE
Use new flag '--stdin-post-extract' for post-install commands

### ISSUE THREAD

**Issue by toluschr** [xdeb-org/xdeb-install#17]
https://github.com/xdeb-org/xdeb/commit/f8d1b589996977ce4f9902469c9b47b1f426593d added a new flag '--stdin-post-extract'.  If specified, `xdeb` executes commands from stdin right after fixing conflicts. This could be helpful in combination with `xdeb-install`'s `post-install` option.

**Comment by thetredev** [comment]
Thanks for the heads up!

Unfortunately my "let's try Arch again" senses kicked in - I just had to try out KDE 6.0 on real hardware lol.

And now I ended up on Linux Mint LMDE of all things.

But development shouldn't be affected by it because of the devcontainer I've setup. It doesn't reflect a complete Void setup but I don't think this is gonna be a problem.

**Comment by thetredev** [comment]
Hmmm.. maybe I should allow the user to pass every argument to `xdeb`, like so:
```
$ xdeb-install <xdeb-install options/flags> -- <xdeb args>
```

@toluschr what do you think? Currently `xdeb-install` is kinda forcing some basic arguments to `xdeb` and that's it.

**Comment by toluschr** [comment]
Would that require `xdeb-install` to parse the `xdeb` arguments? What about using environment variables to configure `xdeb`?

**Comment by thetredev** [comment]
No it would just take everything after `--` and pass it to `xdeb` as is. `xdeb-install` won't do any checking whatsoever and will just print `xdeb`'s output to stdout/stderr.

For environment variables we could just pass all environment variables we have onto `xdeb` as well.

I think we should separate the focus of each project. If we start verifying `xdeb` args from within `xdeb-install`, we could lose track updating both projects with stuff only one needs.

So I vote for giving `xdeb` all we get.

**Comment by toluschr** [comment]
I agree with you on argument parsing. Can the user break `xdeb-install` by passing incorrect arguments to `xdeb`?

**Comment by thetredev** [comment]
I don't think so. Everything after `--` is ignored by `xdeb-install`. I guess there's some shell magic you can hack together to break any tool via `stdin` but that's a thing the shell developers have to deal with I assume.

**Comment by thetredev** [comment]
@toluschr bump
